src/common/websocket/api.ts

```diff
@@ -98,4 +99,16 @@ export default class DerivWS {
             delete this.pendingRequests[reqId]
         }
     }
+
+    private reloadPage = () => {
+        let lastActionTaken = new Date().getTime()
+        const checkLastAction = () => {
+            const now = new Date().getTime()
+            if (now > lastActionTaken) window.location.reload()
+            else lastActionTaken = now
+        }
+        window.addEventListener('mousemove', checkLastAction)
+        window.addEventListener('touchstart', checkLastAction)
+        window.addEventListener('keydown', checkLastAction)
+    }
 }
```

Comment:
Hello @NikitK-deriv,

Is it possible to implement something like `send({ "ping": 1 }, callback)` that is triggered every `1.5s` or so, instead of reloading the page?

You could use something like `setInterval or setTimeout` for it.

Because I see in the API Playground, there is a `ping` endpoint I think that is meant for this purpose.

Please look into it. 🙂  Thank you